ouroboros-consensus/src/Ouroboros/Consensus/Mempool/TxSeq.hs

```diff
@@ -144,3 +148,17 @@ splitAfterTicketNo (TxSeq txs) n =
     case FingerTree.split (\m -> mMaxTicket m > n) txs of
       (l, r) -> (TxSeq l, TxSeq r)
 
+-- | Convert a 'TxSeq' to the type which is expected to be returned from the
+-- 'Mempool' API.
+fromTxSeq :: TxSeq tx -> Seq (tx, TicketNo)
```

Comment:
I agree with Thomas here that we don't want this returning a `Seq`. We can keep this name but have it convert to a list instead.